r/version_5/dm02_data_standardise_v5.R

```diff
@@ -38,8 +41,10 @@ for(r_name in r_names){
    
    ## Read in _1 data
    dt <- qs::qread(input_data)
+   dt <- as.data.table(dt)
+   dt[, panel := as.character(panel)]
+   dt[panel == "Panel G", panel := "Panel C"]
```

Comment:
It fails for me on this? I'm guessing this only works on the kids code and not on the adults?

Reply:
Moved to be conditional